phpstan.neon.dist

```diff
@@ -13,6 +13,12 @@ parameters:
 		-
 			message: '#Return typehint of method KnpU\\OAuth2ClientBundle\\Client\\Provider\\[a-zA-Z0-9\\_]+::fetchUser\(\) has invalid type [a-zA-Z0-9\\_]#'
 			path: ./src/Client/Provider
+		-
+			message: '#Class League\\OAuth2\\Client\\Provider\\[a-zA-Z0-9\\_]+ not found\.#'
+			path: ./src/Client/Provider
+		-
+			message: '#Call to method [a-zA-Z0-9\\_]+\(\) on an unknown class League\\OAuth2\\Client\\Provider\\[a-zA-Z0-9\\_]+\.#'
+			path: ./src/Client/Provider
```

Comment:
PHPStan shows some errors because I added a `logout` method to the `PassageClient`.
https://github.com/knpuniversity/oauth2-client-bundle/actions/runs/6580735273/job/17879341475

To logout from Passage you need to be redirected from a allowed logout url which you define in the backend of Passage when you configure OAUTH.

It would be okay for me to remove it from the `PassageClient` and I will document in `malteschlueter/oauth2-passage` how to logout.